entries/docs/SWC-100.md

```diff
@@ -1,10 +1,23 @@
+# Please note, this content is no longer actively maintained.
+
+The content of the SWC registry has not been thoroughly updated since 2020. It is known to be incomplete and may contain errors as well as crucial ommissions.
```

Comment:
```suggestion
The content of the SWC registry has not been thoroughly updated since 2020. It is known to be incomplete and may contain errors as well as crucial omissions.
```